README.md

```diff
@@ -510,6 +510,7 @@ Table of Contents
   * [checklyhq.com](https://checklyhq.com) - Checkly is the API & E2E monitoring platform for the modern stack: programmable, flexible and loving JavaScript. Generous free tier for devs.
   * [crossbrowsertesting.com](https://crossbrowsertesting.com) - Manual, Visual, and Selenium Browser Testing in the cloud - [free for Open Source](https://crossbrowsertesting.com/open-source)
   * [cypress.io](https://www.cypress.io/) - Fast, easy and reliable testing for anything that runs in a browser. Cypress Test Runner is always free and open source with no restrictions and limitations. Cypress Dashboard is free for open source projects for up to 5 users.
+  * [Cypress Recorder by Preflight](https://cypress.preflight.com/) - Create AI-powered Cypress Tests/POM models on your browser. It's open-source except for the AI part. It's free for up to 50 test creations with Self-healing scripts, Email, and Visual testing.
```

Comment:
I guess `50 Cypress Test Creation Credits` over all time, not per month? If that's the case then I dont think it will fit the list sorry

Reply:
Just pushed an update. I thought it'd be more generous the other way. New signups will get 5 Test Creation free each month.